src/Twig/Extension/PageExtension.php

```diff
@@ -288,10 +262,10 @@ public function controller($controller, $attributes = [], $query = [])
      *
      * @return string
      */
-    private function render($template, array $parameters = [])
+    private function render(Environment $twig, $template, array $parameters = [])
```

Comment:
but @Hanmac I was thinking we do not use render like this

usually we just call 

```php
$this->render('template.html.twig', []);
```

Reply:
this function currently is only called by the `breadcrumb` function.
such breadcrumbs should only just be called once per page load?

if we don't need resources stored, then we could remove the render function and just call:
`$twig->render($template, $parameters)` directly

Reply:
@VincentLanglet your opinion? do we really need to store the twig templates when it most likely only render it once for breadcrumb?

Reply:
I would say that since it was stored before, we should still doing it, in order to avoid to many changes.

Reply:
It could be worth to simplify this code.

Reply:
Can be done in another PR if needed :)